halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/distributed/kubernetes/v2/KubernetesV2DeckService.java

```diff
@@ -70,6 +75,100 @@ public boolean runsOnJvm() {
     }
   }
 
+  @Override
```

Comment:
this is for creating the init containers to download the required resources. Since we're going to use script tags to download resources, we don't need to create init containers / volume mounts for this